decimal64.go

```diff
@@ -176,6 +275,45 @@ func (d Decimal64) parts() (fl flavor, sign int, exp int, significand uint64) {
 	}
 	return
 }
+func (dec decParts) isNan() bool {
+	return dec.fl == flQNaN || dec.fl == flSNaN
+}
+
+// decParts gets the parts and returns in decParts stuct, doesn't get the magnitude due to performance issues
+func (d *Decimal64) getParts() decParts {
```

Comment:
The previous name was fine. Just `parts` would be even better.

soapbox: I'm not a fan of `get…` prefixes, since such functions generally don't mutate their receiver. They don't do something to the subject; they simply tell you something about it. From another perspective, Calling it `getParts` instead of `parts` is like calling the trigonometry function `computeCos` instead of just `cos`.

Reply:
Yes
Although previous name was ambiguous with struct
I will eventually name this function parts but there's still the original parts function that is still a dependency for other functions

Reply:
added TODO